Libplanet/Action/Loader/SimpleActionLoader.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using Bencodex.Types;
+using Libplanet.Action.Sys;
+using Libplanet.Blockchain;
+
+namespace Libplanet.Action.Loader
+{
+    /// <summary>
+    /// An <see cref="IActionLoader"/> implementation for a single <see cref="IAction"/> type.
+    /// </summary>
+    public class SimpleActionLoader : IActionLoader
+    {
+        private Type _type;
+
+        /// <summary>
+        /// Creates a new <see cref="SimpleActionLoader"/> instance.
+        /// The <see cref="IAction"/> type to load is restricted to given type
+        /// <paramref name="type"/>, except for those handled by <see cref="Registry"/>
+        /// for system <see cref="IAction"/>s.
+        /// </summary>
+        /// <param name="type">The type of <see cref="IAction"/> to load.  It should match
+        /// the <see cref="BlockChain{T}"/>'s type parameter.</param>
+        /// <exception cref="ArgumentException">Thrown when given type <paramref name="type"/>
+        /// is not a <see langword="class"/> or cannot be assigned to <see cref="IAction"/>.
+        /// </exception>
+        public SimpleActionLoader(Type type)
+        {
+            if (!type.IsClass)
+            {
+                throw new ArgumentException(
+                    $"Given {nameof(type)} must be a class: {type}", nameof(type));
+            }
```

Comment:
Despite of below check(`typeof(IAction).IsAssignableFrom(type)`), does this check needed?

Reply:
Welp, for one thing, `typeof(IAction)` is assignable to `typeof(IAction)` (i.e. getting rid of it allows something like `new SingleActionLoader(typeof(IAction))`), which is I think is non-sensical. 😅

Only allowing `class` but not `struct` might be a little arbitrary tho. 🤔